app/templates/macros/nav_macros.html

```diff
@@ -73,6 +73,45 @@ <h3 class="ui header">Bulk</h3>
     {% endif %}
 {% endmacro %}
 
+
+{% macro mobile_account_items(current_user) %}
+    <a href="{{ url_for('main.about') }}" class="item">About</a>
+    <a href="{{ url_for('contact.index') }}" class="item">Contact Us</a>
+    {% if current_user.is_authenticated() %}
+        <div class="ui simple dropdown item">
```

Comment:
we're not mobile responsive for admin so not really, but i can make that change and commit it if we want